tests/test_base_script.py

```diff
@@ -248,11 +268,21 @@ def test_non_configurable_script_get_schema(self):
                 await script.do_configure(configure_data)
             self.assertEqual(script.state.state, ScriptState.UNCONFIGURED)
 
-            # now real configuration
+            # now real configuration; specify nonexistent checkpoints
+            # just to test that the configure command handles checkpoints
```

Comment:
Re-reading this text, you say "specify nonexistent checkpoints", but the first call to `configure_script` seems to have legit ones?